distribution/docs/src/main/jdocs/content/_configuring/hardening-intro-contents.adoc

```diff
@@ -0,0 +1,15 @@
+:title: Hardening Intro
+:type: configuringIntro
+:status: published
+:summary: Introduction to Hardening.
+:order: 03
+
+.Security Hardening
+****
```

Comment:
❓ should more of these sections have this background? I see it on `Security Hardening` and `Admin Console Tutorial`?